File: problems/palindromes/palindrome.swift
let words = [" Madam", "Nurses Run", " Palinwho?"]

func returnPalindromes(_ words: [String]) -> [String] {
    return words.filter { isPalindrome($0) }
}

private func isPalindrome(_ word: String) -> Bool {
  let word = word.lowercased().split(separator: " ").joined(separator: "")
    return word == String(word.reversed())
}

let palindromes = returnPalindromes(words)
print(palindromes)

File: problems/palindromes/palindrome.py
def main():
  words = [" Madam", "Nurses Run", " Palinwho?"]
  palindromes = returnPalindromes(words)
  print(palindromes)

def returnPalindromes(words):
    return list(filter(lambda x: isPalindrome(x), words))

def isPalindrome(word):
    word = word.lower().replace(" ", "")
    return word == word[::-1]

main()
